pack.go

```diff
@@ -126,6 +128,11 @@ var mediaTypeRegexp = regexp.MustCompile(`^[A-Za-z0-9][A-Za-z0-9!#$&-^_.+]{0,126
 //
 // artifactType and opts.ConfigDescriptor.MediaType MUST comply with RFC 6838.
 //
+// Each time when PackManifest is called, it generates a new time stamp in the
+// manifest annotations with the key "org.opencontainers.image.created". To make
+// PackManifest reproducible, set "org.opencontainers.image.created" with a value
```

Comment:
Same here for consts.

Reply:
This comment is a duplicate of https://github.com/oras-project/oras-go/pull/749/files#r1575970704

Reply:
Replaced the const with the variable name.